styleguide/source/_patterns/02-molecules/search-result/search-result~as-best-bet.json

```diff
@@ -0,0 +1,29 @@
+{
+  "searchResult": {
+    "image": {
+      "alt": "alt text",
+      "src": "https://ipsumimage.appspot.com/800x600x186?l=3:2|800x600&s=36",
```

Comment:
@JaeBea The image that I'm using for the best bet is 800x600.

Reply:
@labbydev I am requesting that this is changed to match what is on the subcategory search area, 566x370. I have put a screenshot below.
![image](https://user-images.githubusercontent.com/4438120/44594508-fc4d0980-a78b-11e8-9fcc-e38c3a7f8c43.png)

Reply:
Done